sys/include/net/gnrc/ipv6/nib/pl.h

```diff
@@ -122,6 +125,42 @@ void gnrc_ipv6_nib_pl_del(unsigned iface,
 bool gnrc_ipv6_nib_pl_iter(unsigned iface, void **state,
                            gnrc_ipv6_nib_pl_t *ple);
 
+#if IS_USED(MODULE_EVTIMER) || defined(DOXYGEN)
+/**
+ * @brief   Provides the time difference in milliseconds until the prefix list
+ *          entry is valid
+ *
+ * @param[in] ple   A prefix list entry.
+ *
+ * @return  The time difference in milliseconds until the prefix list entry
+ *          is valid. UINT32_MAX if it is valid forever.
+ */
+static inline uint32_t gnrc_ipv6_nib_pl_valid_offset(const gnrc_ipv6_nib_pl_t *ple)
+{
+
+    return (ple->valid_until == UINT32_MAX)
+        ? UINT32_MAX
+        : (ple->valid_until - evtimer_now_msec());
+}
+
+/**
+ * @brief   Provides the time difference in milliseconds until the prefix list
```

Comment:
```suggestion
 * @brief   Provides the time in milliseconds for which the prefix list
```
?

"until the prefix list entry is preferred" is confusing - to me it sounds like it would only be preferred *after* this time has elapsed.